pkg/controllers/user/nodesyncer/nodessyncer.go

```diff
@@ -196,7 +196,7 @@ func (m *nodesSyncer) updateAnnotations(node *corev1.Node, obj *v3.Node, nodePla
 
 	node, obj = node.DeepCopy(), obj.DeepCopy()
 	node.Annotations = finalMap
-	obj.Spec.MetadataUpdate.Labels = v3.MapDelta{}
+	obj.Spec.MetadataUpdate.Annotations = v3.MapDelta{}
```

Comment:
Is it an issue that we are removing the labels update?

Reply:
No it was a typo, we have a separate function which handles labels